Sources/Client/NetClient.cpp

```diff
@@ -752,14 +752,12 @@ namespace spades {
 		std::string NetClient::DisconnectReasonString(enet_uint32 num) {
 			switch (num) {
 				case 1: return _Tr("NetClient", "You are banned from this server.");
-				case 2:
-					// FIXME: this number seems to be used when per-IP connection limit was
-					// exceeded.
-					//        we need to check other usages
-					return _Tr("NetClient", "You were kicked from this server.");
+				case 2: return _Tr("NetClient", "You have too many connections to this server.");
```

Comment:
The word “you” might be inappropriate here because it’s not necessarily “you” who has connections (e.g., other people behind a NAT).

Reply:
I had similar concerns but I was unsure how to phrase it instead. "Your IP" might be correct at the moment, but:
 - people, in general, are not that familiar with where their IP comes from, especially in larger networks and with the increased prevalence of CGNATs
 - this is a technical detail and might change in the future

Perhaps "Your location" or "Your network" is a good solution?

Reply:
Seems like "Your network's public address has too many connections to this server" would be the best solution, even though some users may not understand it, I believe it's best to deliver the most accurate message.

Maybe a text below with additional info could be added including the common reasons this might be happening.